### PR TITLE
leap: fixes #3278

### LINKED ISSUE #3278: search: input must be lowercase
**Describe the bug**
If you type "General" rather than "general" the search box it doesn't find the channel. It seems input must be entirely lowercase for it to produce results

**To Reproduce**
1. ctrl+/
2. type "General"
3. find nothing
4. type "general"
5. find the channel

**Expected behavior**
search input should be case insensitive

**Desktop (please complete the following information):**
 - OS: arch linux 64-bit
 - Browser: qutebrowser
 - Base hash: dg0qj

### PR DESCRIPTION
**Pull request by tylershuster**
Fixing a few bugs that shipped with leap. On the list:

- [x] #3277
- [x] #3278
- [x] #3279
- [ ] #3280
- [ ] bonus: #3274 

I'll watch those issues to see if anything develops

### CODE REVIEW

File: pkg/interface/src/components/Omnibox.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ import OmniboxResult from './OmniboxResult';
 import { cite } from '../lib/util';
 
 export class Omnibox extends Component {
+  searchCategories = ['commands', 'apps', 'groups', 'subscriptions'];
```
Comment (tylershuster) [inline pkg/interface/src/components/Omnibox.js:12]: DRY so order can be changed or appended

**Comment by matildepark** [comment]
lol I'm doing these very same fixes Tyler.

**Comment by matildepark** [comment]
I actually have all but #3274 written and committed, just writing proper autojoin logic to catch errors.

**Comment by tylershuster** [comment]
I didn't think you'd even be up.

**Pull request closed without merging**